lib/config/rules/functionality.js

```diff
@@ -72,7 +72,7 @@ module.exports = {
   // Require a `default` case in all `switch` statements.
   'switch-default': false,
   // Requires `===` and `!==` in place of `==` and `!=`.
-  'triple-equals': true,
+  'triple-equals': [true, 'allow-null-check'],
```

Comment:
Cool.  This is good to go once everything not directly related to `triple-equals` is removed from this PR.